examples/knowledge_tuning/knowledge_tuning_with_reasoning_model/reasoning_sdg_financebench.py

```
@@ -0,0 +1,582 @@
+        "DATA_BASE_PATH env-var is required – point it to your dataset root."
+    )
+
+# %% [markdown]
+# ### Intro
+# - For this reasoning demo we will use the financeBench documents to customize a model on SEC fillings
+
+# %%
+from datasets import load_dataset, concatenate_datasets
+from openai import OpenAI
+
+endpoint = f"http://localhost:8000/v1"
```

Flake8 (7.2.0)

[error] f-string is missing placeholders

(F541)